core/src/main/scripts/importer/validateData.py

```diff
@@ -249,7 +249,7 @@ class Validator(object):
     REQUIRE_COLUMN_ORDER = True
     ALLOW_BLANKS = False
 
-    def __init__(self, study_dir, meta_dict, portal_instance, logger):
+    def __init__(self, study_dir, meta_dict, portal_instance, logger, relaxed_mode):
```

Comment:
Also, it would be a good idea to add a function that a study without clinical attribute definitions in `core/src/test/scripts/unit_tests_validate_data.py`, as a regression test for the new option.